npm/private/lifecycle/lifecycle-hooks.js

```diff
@@ -151,12 +151,14 @@ async function main(args) {
 // Copy contents of a package dir to a destination dir (without copying the package dir itself)
 async function copyPackageContents(packageDir, destDir) {
     const contents = await fs.promises.readdir(packageDir)
-    contents.forEach(async (file) => {
-        await copyRecursive(
-            path.join(packageDir, file),
-            path.join(destDir, file)
-        )
-    })
+    await Promise.all(
+        contents.map(async (file) => {
```

Comment:
I think we can just do:
```
await Promise.all(contents.map(file => copyRecursive(...)))
```
Since `Promise.all` handles all the async we just need to pass an array of promises...

Reply:
Nice catch 👍